rsa.go

```diff
@@ -43,7 +44,17 @@ func (pk *RsaPublicKey) Raw() ([]byte, error) {
 }
 
 func (pk *RsaPublicKey) Encrypt(b []byte) ([]byte, error) {
-	return rsa.EncryptPKCS1v15(rand.Reader, pk.k, b)
+	partLen := pk.k.N.BitLen()/8 - 11
+	chunks := split([]byte(b), partLen)
+	buffer := bytes.NewBuffer([]byte(""))
```

Comment:
that's incorrect usage of the buffer -- just pass nil or create a new buffer object directly.
See the docs in https://godoc.org/bytes